### ISSUE TITLE
Document procedure for manually submitting and attaching large files

### ISSUE THREAD

**Issue by RachelDCE** [curationexperts/laevigata#2080]
**ACCEPTANCE**

When a student has a supplemental file larger than 100 MB

- [ ] We have a written procedure for submitting and tracking the file (Emory Scholarly Communications)
  - [x] DCE would like a FreshDesk ticket with 
       * Student's name & login ID
       * Thesis title & ID
       * Link to file (DCE agent who picks up the ticket will request access to open and download file)

- [x] Look for prior art on how to manually attach a file to an ETD
  - [x] https://github.com/curationexperts/laevigata/wiki/How-to-manually-attach-a-file-to-an-ETD
- [x] We have a written procedure for attaching the file to the ETD (DCE)
- [x] We have tested attaching a large file using the procedure 
  - [x] supplementary file added to https://qa-etd.library.emory.edu/concern/etds/zg64tn32f?locale=en

**Comment by maxkadel** [comment]
Note: File transfer over scp from local computer up to server (current directions) is pretty slow - ~20 min for a 3 GB text file, and your computer can't go to sleep while it's in progress or you have to start over.

**Comment by mark-dce** [comment]
If the transfer takes that long, then yes, `scp` might not be the best option since I think it's generally not resumable if it fails mid-point.

OPTIONS:
* SFTP is a pretty friendly alternative, but it requires an FTP service to be running on the host (which raises additional security config concerns).
* RSYNC might be an alternative that allows resuming partially complete transfers

**Comment by RachelDCE** [comment]
I worked with Kylie and verified that the workflow we outlined aligns with her understanding. We successfully tested sharing/sending a file in OneDrive - and receiving/downloading file. Her organization has prevented the option for "Anyone with link can....", so she does have to share with specific email addresses. 

Options: 
- [ ] We could give her a list of emails (which we need to keep updated for her) -or -
- [ ] *We could set up or reuse existing email like we have with: contact@curationexperts.com or finance@curationexperts.com
